src/ResourceManager/RecoveryServices.Backup/Commands.RecoveryServices.Backup.Providers/Providers/AzureSqlPsBackupProvider.cs

```diff
@@ -204,9 +204,8 @@ public AzureRmRecoveryServicesBackupSchedulePolicyBase GetDefaultSchedulePolicyO
         public AzureRmRecoveryServicesBackupRetentionPolicyBase GetDefaultRetentionPolicyObject()
         {
             AzureRmRecoveryServicesBackupSimpleRetentionPolicy defaultRetention = new AzureRmRecoveryServicesBackupSimpleRetentionPolicy();
-            defaultRetention.RetentionDuration = new Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.Models.RetentionDuration();
-            defaultRetention.RetentionDuration.RetentionDurationType = Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.Models.RetentionDurationType.Days;
-            defaultRetention.RetentionDuration.RetentionCount = 180;
+            defaultRetention.RetentionDurationType = Microsoft.Azure.Commands.RecoveryServices.Backup.Cmdlets.Models.RetentionDurationType.Months;
```

Comment:
Should this be Months or Years? I'm not sure if this is true, but the PR title seems to be contradicting with this change. Please check.